### ISSUE TITLE
Unable to connect to web socket error

### ISSUE THREAD

**Issue by Dryskol19** [Ylianst/MeshCentral#3875]
I just updated from v1.0.3 to v1.0.4 on a debian server running one instance of Mesh Central and another Windows 10 box running a separate instance. Separate networks as well. Update completed but after logging in I am prompted with an Unable to connect to web socket error, click to reconnect loop. Curious if anyone else encountered the same
![Error](https://user-images.githubusercontent.com/103533456/163039253-2c966cd2-921f-43c9-ba44-4b2c20ea166b.jpg)
.

**Comment by Dryskol19** [comment]
Just a a small update, tried incognito mode in exiting browser (Edge) and different browser (Firefox) and can now access the Windows 10 box instance no issue. Still getting socket error when trying to access Debian instance.

**Comment by Ylianst** [comment]
In the settings section of the config.json, add:

```
"CookieIpCheck": false
```

Let me know if that fixes it and what value you had before.

Thanks.

**Comment by Dryskol19** [comment]
Added that value to the Windows 10 instance .json, restarted Mesh Central Service. Now I get the web socket error regardless of current browsing session or incognito mode. JSON is below.
![JSON](https://user-images.githubusercontent.com/103533456/163045161-ced93e9c-e67b-4800-8c42-8304f95d6067.jpg)

**Comment by Ylianst** [comment]
Just rolled back "latest" to v1.0.3 and investigating this now!

**Comment by Fefo60** [comment]
Howto rollback?

**Comment by Ylianst** [comment]
you can do:

```
npm install meshcentral@1.0.3
```

I am getting the same issue, investigating it now!!!

**Comment by Ylianst** [comment]
Once I get a fix for this, I will send it out asap.

**Comment by Ylianst** [comment]
Just published MeshCentral v1.0.5 with a fix for this. If you are stuck, replace `node_modules/meshcentral/webserver.js` with the [file found on GitHub here](https://raw.githubusercontent.com/Ylianst/MeshCentral/master/webserver.js).

Another way to fix this is to edit `node_modules/meshcentral/webserver.js` and replace this line:

```
if ((req.session.ip != null) && (req.clientIp != null) && checkCookieIp(req.session.ip, req.clientIp)) { req.session = {}; }
```

With this line:

```
if ((req.session.ip != null) && (req.clientIp != null) && !checkCookieIp(req.session.ip, req.clientIp)) { req.session = {}; }
```

Note the ! is missing on the first line.
